chain/network/src/peer/peer_actor.rs

```diff
@@ -1404,13 +1401,17 @@ impl actix::Handler<WithSpanContext<Stop>> for PeerActor {
 
     #[perf]
     fn handle(&mut self, msg: WithSpanContext<Stop>, ctx: &mut Self::Context) -> Self::Result {
-        let span =
-            tracing::trace_span!(target: "network", "handle", handler = "Stop", actor="PeerActor")
-                .entered();
-        span.set_parent(msg.context);
+        let span = tracing::trace_span!(
+                target: "network",
+                "handle",
+                handler = "Stop",
+                actor = "PeerActor")
+        .entered();
+        let WithSpanContext { msg, context, .. } = msg;
+        span.set_parent(context);
```

Comment:
```suggestion
    let (_span, msg) = handler_span!("network", "PeerActor", "Stop", msg);
```

Reply:
The macro defines a `tracing::debug_span!` and this is a `trace_span!`. PeerActors are stopped rarely, should be fine upgrading it to a `debug_span!`.